internal/schema/schema.go

```diff
@@ -395,57 +418,87 @@ func resolveInputObject(s *Schema, values common.InputValueList) error {
 	return nil
 }
 
-func parseSchema(s *Schema, l *common.Lexer) {
+func parseSchema(s *Schema, l *common.Lexer) error {
```

Comment:
I think we should keep error returns to a single form here for now—that is, errors are returned via `panic` (which is strange, yes). I believe the intention was to prevent error checking on every `l.ConsumeX()` call.

Either way, I'm not sure that introducing another error path is something we want to do in the short-to-medium term.